src/com/oltpbenchmark/api/Worker.java

```diff
@@ -220,6 +220,17 @@ synchronized public void cancelStatement() {
         }
     }
 
+    public void closeConnection() {
+        try {
+            if(ll_conn != null) {
+                ll_conn.close();
+                LOG.info("Closing Persistent connections");
```

Comment:
Not needed. Will generate lot of logs

Reply:
This is just for debugging purpose. Will remove it before merge